views/js/pciCreator/dev/adaptiveChoiceInteraction/test/commonRenderer.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=EDGE" />
+        <title>PCI Text Reader Interaction Common Renderer Test</title>
+        <link rel="stylesheet" type="text/css" href="/tao/views/js/lib/qunit/qunit.css">
+        <script type="text/javascript" src="/tao/views/js/lib/require.js"></script>
+        <script type="text/javascript" src="/tao/views/js/lib/qunit/qunit.js"></script>
+
+        <link rel="stylesheet" type="text/css" href="/tao/views/css/tao-main-style.css" />
+        <link rel="stylesheet" type="text/css" href="/taoQtiItem/views/css/qti.css" />
+        
+        <script  type="text/javascript">
+            QUnit.config.autostart = false;
+            require(['/tao/ClientConfig/config'], function(){
+                require(['taoQtiItem/qtiCommonRenderer/test/runner'], function(runner){
+                    QUnit.start();
+
+                    runner.run({
+                        relBaseUrl : 'qtiItemPci/pciCreator/dev/adaptiveChoiceInteraction/test/samples/',
+                        runtimeLocations : {
+                            adaptiveChoiceInteraction : 'qtiItemPci/pciCreator/dev/adaptiveChoiceInteraction/'
+                        },
+                        assertions : {
+                            interactionCount : 1,
+                            responses : []
+                        }
+                    });
+
```

Comment:
Unfortunately, that level of unit testing we used to have is no longer sufficient. Now, we need all the functionnalities, possible events, get/setResponse, get/setState  to be fully tested, at least for the renderer of new interactions.
You can have a more up-to-date example of test in here : https://github.com/oat-sa/extension-tao-itemqti/blob/master/views/js/test/qtiCommonRenderer/interactions/choice/test.js